clustered/client/src/test/java/org/ehcache/clustered/client/internal/store/operations/ChainResolverExpiryTest.java

```diff
@@ -81,6 +81,7 @@ public void testGetExpiryForAccessIsIgnored() {
     verify(expiry, times(1)).getExpiryForUpdate(anyLong(), any(ValueSupplier.class), anyString());
 
     assertThat(resolvedChain.isCompacted(), is(true));
+    assertThat(resolvedChain.getCompactedSize(), is(2));
```

Comment:
I recommend against duplicating that assertion in all the places. Let's have one set of tests that verifies it behaves as expected and assume it is correct for others.